src/applications/vaos/components/calendar/CalendarCell.jsx

```diff
@@ -79,6 +80,13 @@ const CalendarCell = ({
   const momentDate = moment(date);
   const dateDay = momentDate.format('D');
   const ariaDate = momentDate.format('dddd, MMMM Do');
+  const buttonLabel = inSelectedArray
+    ? `${ariaDate}. ${
+        renderSelectedLabel
+          ? renderSelectedLabel(date, selectedDates)
+          : 'Date selected.'
```

Comment:
Based on my reading of the ticket, should this just be `Selected.` instead of `Date selected`? Although, I personally think `Date selected.` is clearer and Trevor might be ok with it as is.

Reply:
Oh, good catch. I think I got "date selected" stuck in my head because of the issue title, but that's not what the title meant. I think it reads more concisely the way it is in the ticket, so I'll just switch to that.